### PR TITLE
[Images] Upload via URL feature

### PR DESCRIPTION
**Pull request by marciocloudflare**
Adresses [PCX-3632](https://jira.cfops.it/browse/PCX-3632) and [PCX-3510](https://jira.cfops.it/browse/PCX-3510)

### CODE REVIEW

File: content/images/cloudflare-images/upload-images/upload-via-url.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+```
+
+If your origin server returns an error while fetching the images, the API response will return a `4xx` error.
+
+Refer to [Upload an image using a single HTTP request](https://api.cloudflare.com/#cloudflare-images-upload-an-image-using-a-single-http-request) for more information.
+
+{{<Aside type="note" header="Note">}}
+
+The `--form 'file='` and `--form 'url='` fields in the API documentation example are mutually exclusive.
```
Comment (xjewer) [inline content/images/cloudflare-images/upload-images/upload-via-url.md]: I'd remove this:  _in the API documentation example_

Comment (marciocloudflare) [reply content/images/cloudflare-images/upload-images/upload-via-url.md]: Thanks!

Comment (marciocloudflare) [reply content/images/cloudflare-images/upload-images/upload-via-url.md]: done

File: content/images/cloudflare-images/upload-images/_index.md [hunk 2]
```diff
@@ -6,15 +6,16 @@ weight: 3
 
 # Upload images
 
-Cloudflare Images allows developers to upload images that cover a wide range of use cases.
+Cloudflare Images allows developers to upload images in different manners, for a wide range of use cases.
```
Comment (deadlypants1973) [inline content/images/cloudflare-images/upload-images/_index.md]:
```suggestion
Cloudflare Images allows developers to upload images using different methods, for a wide range of use cases.
```

File: content/images/cloudflare-images/upload-images/upload-via-url.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+weight: 4
+meta:
+  title: Upload via URL
+---
+
+# Upload via URL
+
+Sometimes it can be useful to use a URL of an image instead of uploading its data. To accommodate this need, Cloudflare Images provides an option to use a URL to migrate images to Cloudflare without fetching them first.
+
+Refer to [Supported image formats](/images/cloudflare-images/upload-images/supported-formats) to learn more about the supported image formats you can upload.
```
Comment (deadlypants1973) [inline content/images/cloudflare-images/upload-images/upload-via-url.md]:
```suggestion
To learn more about the supported image formats you can upload, refer to [Supported image formats](/images/cloudflare-images/upload-images/supported-formats).
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**